Lesson3.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+public class Lesson3
+{
+	public Lesson3()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace all tab characters in this file by sequences of white-spaces.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4971681)